### PR TITLE
Dorian javadoc

### PR DESCRIPTION
**Pull request by DorianRec**

### CODE REVIEW

File: pom.xml [hunk 1]
```diff
@@ -74,6 +74,11 @@
             <version>4.11</version>
             <classifier>javadoc</classifier>
         </dependency>
+        <dependency>
+            <groupId>io.reactivex.rxjava2</groupId>
+            <artifactId>rxjava</artifactId>
+            <version>2.1.10</version>
+        </dependency>
```
Comment (jotoh98) [inline pom.xml:81]: why is this here? just to suffer?

**Review by jotoh98 — CHANGES_REQUESTED** [review]

File: pom.xml (hunk 1 above)
Comment (jotoh98) [reply pom.xml:81]: hello from danielle

Comment (DorianRec) [reply pom.xml:81]: Added this for Java CL running the tests
